2d/Assets/obstacle_cntr.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class obstacle_cntr : MonoBehaviour
```

Comment:
controlは普通ctrlと略します。
これを修正するついでに、キャメルケースで書いてみませんか？C#はキャメルケースで書いた方が標準と統一感があって好ましく感じませんか？
```suggestion
public class ObstacleCtrl : MonoBehaviour
```